common/src/main/java/com/microsoft/identity/common/internal/ui/webview/certbasedauth/AbstractSmartcardCertBasedAuthChallengeHandler.java

```diff
@@ -107,20 +107,13 @@ public void onGetSession(@NonNull final ISmartcardSession session) throws Except
                     onPausedSmartcardDiscovery();
                     return;
                 }
-
+                final ICancelCbaCallback cancelCallback = getGeneralCancelCbaCallback(request);
```

Comment:
I create an object here instead of calling `getGeneralCancelCbaCallback` directly as a parameter in `showCertPickerDialog` in order to reduce the amount of merge conflicts I'll get in the original feature work PR :)